images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -323,4 +325,4 @@ if (-not (Test-Path $OutputDirectory)) { New-Item -Path $OutputDirectory -ItemTy
 #
 Write-Host $markdownExtended
 $systemInfo | Out-File -FilePath "${OutputDirectory}/systeminfo.txt" -Encoding UTF8NoBOM
-$markdown | Out-File -FilePath "${OutputDirectory}/systeminfo.md" -Encoding UTF8NoBOM
\ No newline at end of file
+$markdown | Out-File -FilePath "${OutputDirectory}/systeminfo.md" -Encoding UTF8NoBOM

```

Comment:
Could you please get rid of these unintended changes?

Reply:
Sure, that is editor config issue. :(